packages/sdk-platform/src/interfaces/Items.ts

```diff
@@ -7,7 +7,7 @@ import type {
   WithCommonOptions
 } from '@spree/core-api-v2-sdk'
 
-export interface PageAttr {
+export interface ItemAttr {
```

Comment:
Since that's an exported interface, this is a breaking change. 
Would it be possible to avoid changing its name, or create an alias, so that both work (with the old one marked as deprecated?)